curriculum/challenges/english/02-javascript-algorithms-and-data-structures/basic-javascript/finding-a-remainder-in-javascript.md

```diff
@@ -13,12 +13,20 @@ The <dfn>remainder</dfn> operator `%` gives the remainder of the division of two
 
 **Example**
 
-<blockquote>5 % 2 = 1 because<br>Math.floor(5 / 2) = 2 (Quotient)<br>2 * 2 = 4<br>5 - 4 = 1 (Remainder)</blockquote>
+```js
+const five = 5;
+const two = 2;
+const quotient = Math.floor(5 / 2);
+console.log(quotient); // logs 2
+```
```

Comment:
I haven't added `ltr` direction to the `pre` element, because I didn't want to affect something that isn't in the description.

Unfortunately, it follows the direction of the document, so it still needs to be a codeblock.

```js
	5 % 2 = 1
	5 / 2 = 2 // remainder 1
	2 * 2 = 4
	5 - 4 = 1
```